examples/jsm/controls/OrbitControls.js

```diff
@@ -581,17 +596,88 @@ class OrbitControls extends EventDispatcher {
 
 		}
 
-		function handleMouseWheel( event ) {
+		function getTotalOffsetLeftAndTop(dom){
 
-			if ( event.deltaY < 0 ) {
+			let left = 0, top = 0;
+			getOffset(dom);
+
+			function getOffset(dom){
+				left += dom.offsetLeft;
+				top += dom.offsetTop;
+				if(dom.offsetParent){
+					getOffset(dom.offsetParent)
+				}
+			}
+
+			return {
+				top: top,
+				left: left
+			}
+		}
+
+		function followZoom(event){
+
+			let x = (event.offsetX / scope.domElement.clientWidth) * 2 -1,
+			y = -(event.offsetY / scope.domElement.clientHeight) * 2 + 1,
+			v = new Vector3(x, y, 0);
+
+			v.unproject(scope.object);
+			v.sub(scope.object.position).setLength(scope.zoomSpeed * 10) // it needs more speed here;
+			
+			if(scope.object.type === "PerspectiveCamera"){
```

Comment:
I think `scope.objec.isOrthographicCamera` and `scope.objec.isPerspectiveCamera` can be used instead of `scope.object.type`

Reply:
okay I agree, it will be more concise